src/test/java/com/hyperlocal/server/ShopControllerTest.java

```diff
@@ -38,15 +41,19 @@
   private static final String SHOP_INSERT_STATEMENT = "INSERT INTO `Shops` (`ShopName`, `TypeOfService`, `Latitude`, `Longitude`, `AddressLine1`, `MerchantID`) VALUES (?,?,?,?,?,?);";;
```

Comment:
Can you reuse Strings from Constants and avoid declarations in Tests

Reply:
Working on it. Thanks a lot!!!